tests/test/kube.rs

```diff
@@ -318,12 +325,36 @@ impl KubeClient {
             }
         }
 
-        Err(anyhow::anyhow!(
+        Err(anyhow!(
             "Pod condition [{}] was not satisfied within {} seconds",
             condition_type,
             timeout_secs
         ))
     }
+
+    /// Returns the logs for the given pod.
+    pub async fn get_logs(&self, pod: &Pod, tail_lines: Option<i64>) -> Result<Vec<String>> {
+        let pods: Api<Pod> = Api::namespaced(self.client.clone(), &self.namespace);
+
+        let log_params = LogParams {
```

Comment:
Would it maybe make sense exposing LopParams to the tests in the api so that we do not need to change it later on if we support additional parameters?

Reply:
I think this is a good idea. The only downside is that a developer could set `follow` to `true` which makes a test case run forever.

Reply:
Which might be a valid test case, if the test incorporates a timer and stops itself after some processing is finished - or am I thinking too simple here?

Reply:
This is a valid test case but it would make more sense if the echo-server would send messages triggered by the test case and the test case would follow this message stream and check it. But as the echo-server is currently only able to ouput some messages immediately, following the stream is like waiting for nothing, so I refrained from implementing this test case.

Reply:
Ah, I see!
In that case we could panic if follow is set to true for now, until the echo-server is changed... then again, not really necessary I think. I'm fine either way.